backend/cmd/plugins.go

```diff
@@ -184,4 +200,10 @@ func addPluginRoutes(config *HeadlampConfig, r *mux.Router) {
 	}
 
 	r.PathPrefix("/plugins/").Handler(pluginHandler)
+
+	if config.staticPluginDir != "" {
+		staticPluginsHandler := http.StripPrefix(config.baseURL+"/static-plugins/",
```

Comment:
Probably we need similar no-cache code to above for this staticPluginsHandler too?

This part I mean:
```go
	if !config.useInCluster {
		pluginHandler = serveWithNoCacheHeader(pluginHandler)
	}
```

Reply:
I left that out on purpose because these are supposed to be static. So we ship it once and don't care if those contents have changed. If it turns out it's needed later, we can add it, but now I don't see a need.